console/client/src/layout/IDELayout.tsx

```diff
@@ -133,8 +145,13 @@ export function IDELayout() {
           </Tab.Group>
         </div>
         <SidePanel />
-        <RequestModal />
       </div>
+      {invalidTabMessage && <Notification
+        color='text-red-400'
+        icon={<InformationCircleIcon className='flex-shrink-0 inline w-4 h-4 mr-3' />}
+        title='Info Alert!'
+        message={invalidTabMessage}
+      />}
```

Comment:
Is there a good way for us to test this out? I tried typing in an invalid tab name, but I didn't see an error just an empty tab:
<img width="1360" alt="Screenshot 2023-08-25 at 9 27 58 AM" src="https://github.com/TBD54566975/ftl/assets/51647/d9a9e43c-9c59-480f-ad44-6639da0f0ce0">

Reply:
This is what I get with `active-tab=payment` looks like the tab is kind of floating here:
<img width="1360" alt="Screenshot 2023-08-25 at 9 30 55 AM" src="https://github.com/TBD54566975/ftl/assets/51647/87af5f8a-5b13-45f6-9dfa-1d5ab815b347">

Reply:
> ? I tried typing in an invalid tab name, but I didn't see an error just an empty tab:

I showed how in the vide

Reply:
As for payment I can't actually get that module to work locally

Reply:
Okay so you image indicates you were not on commit https://github.com/TBD54566975/ftl/pull/320/commits/f2494dfac55b388a0dc64b3b531b60dc10b913a9

Which was pushed well before you comment. I'd recommend pull down the most recent version and trying that [url](http://localhost:8892/?module=payment&active-tab=payment) again https://github.com/TBD54566975/ftl/pull/320/commits/f2494dfac55b388a0dc64b3b531b60dc10b913a9 which will do nothing nor the weird render thing I can't recreate. 

A link that will error correctly is this http://localhost:8892/?module=payment&tab-type=verb which is missing tab-id

Reply:
Yup, that makes sense since this commit didn't exist yet when I tested this. My comment was made before https://github.com/TBD54566975/ftl/commit/f2494dfac55b388a0dc64b3b531b60dc10b913a9 was done. 

I think there might be one remaining check on whether the URL is valid that should be added. Here's the URL I tested with `http://localhost:8892/?module=currency&tab-id=bogus.convert&tab-type=verb` and `http://localhost:8892/?module=currency&tab-id=currency.bogus&tab-type=verb` and the result is a blank tab with no error message.

![Screenshot 2023-08-27 at 1 34 02 PM](https://github.com/TBD54566975/ftl/assets/51647/b5f7c7f8-4d9e-4756-8081-23a7111dae03)